src/patternfly/components/Switch/switch.scss

```diff
@@ -19,25 +19,28 @@
   // Switch toggle
   --pf-c-switch__toggle--BackgroundColor: var(--pf-global--disabled-color--200);
   --pf-c-switch__toggle--BorderRadius: var(--pf-c-switch--Height);
-  --pf-c-switch__toggle--before--Top: calc(#{var(--pf-c-switch--Height)} / 2 * (1 - 1 / #{var(--pf-global--golden-ratio)}));
+  --pf-c-switch__toggle--before--Top: calc((var(--pf-c-switch--Height) - var(--pf-c-switch__toggle--before--Height)) / 2);
   --pf-c-switch__toggle--before--Left: var(--pf-c-switch__toggle--before--Top);
-  --pf-c-switch__toggle--before--Width: calc(#{var(--pf-c-switch--Height)} / #{var(--pf-global--golden-ratio)});
+  --pf-c-switch__toggle--before--Width: calc(var(--pf-c-switch__label--FontSize) - #{pf-size-prem(2px)});
```

Comment:
It's worth noting that `--pf-global--FontSize--sm` would work here, too. That's what I had in the original PR, and it's the same value used for `--pf-c-switch__toggle-icon--FontSize`. Though I think this is better since it resizes the toggle relative to the font size used in the switch instead of needing to be adjusted manually if you change the switch font size. That said, I think we should update the `--pf-c-switch__toggle-icon--FontSize` var, too, since it's the only thing that doesn't scale when you change the switch font size.

Reply:
Updated the font-size of the icon to be relative to the switch's font-size. However, we might also need to make the padding-left on the icon, relative to the icon size too..

<img width="183" alt="Screen Shot 2019-07-17 at 4 10 14 PM" src="https://user-images.githubusercontent.com/20118816/61408203-7e1ab980-a8ad-11e9-8dda-422a5289c745.png">

Reply:
I tested the value 0.4 and it seems to be consistent across all font-sizes in the switch:

<img width="254" alt="Screen Shot 2019-07-17 at 4 16 08 PM" src="https://user-images.githubusercontent.com/20118816/61408698-9fc87080-a8ae-11e9-82b8-f799fc8a366c.png">
<img width="208" alt="Screen Shot 2019-07-17 at 4 16 26 PM" src="https://user-images.githubusercontent.com/20118816/61408699-9fc87080-a8ae-11e9-84cf-1f2b08e7482f.png">
<img width="246" alt="Screen Shot 2019-07-17 at 4 17 58 PM" src="https://user-images.githubusercontent.com/20118816/61408700-9fc87080-a8ae-11e9-8dd8-49de11cbf50c.png">
<img width="169" alt="Screen Shot 2019-07-17 at 4 18 16 PM" src="https://user-images.githubusercontent.com/20118816/61408701-9fc87080-a8ae-11e9-9ca4-3207bd33a1f3.png">
<img width="156" alt="Screen Shot 2019-07-17 at 4 18 37 PM" src="https://user-images.githubusercontent.com/20118816/61408702-a0610700-a8ae-11e9-8b7e-8bf146e452fe.png">
<img width="150" alt="Screen Shot 2019-07-17 at 4 18 58 PM" src="https://user-images.githubusercontent.com/20118816/61408703-a0610700-a8ae-11e9-9a9f-69bc6de22bf8.png">